FEXCore/Source/Interface/Core/JIT/Arm64/JIT.cpp

```diff
@@ -791,6 +791,10 @@ CPUBackend::CompiledCode Arm64JITCore::CompileCode(uint64_t Entry,
   adr(TMP1, &JITCodeHeaderLabel);
   str(TMP1, STATE, offsetof(FEXCore::Core::CPUState, InlineJITBlockHeader));
 
+  // Trigger a fault if there are any pending interrupts
+  strb(ARMEmitter::XReg::zr, STATE, offsetof(FEXCore::Core::InternalThreadState, InterruptFaultPage) -
```

Comment:
Maybe wrap this in WIN32 checks until the Linux side catches up to the ability to pause a thread, and also vetting the performance impact on that side (should fall in to noise for one cycle store but would be nice to validate)